src/commonControls/LanguageStrings.ts

```diff
@@ -186,7 +153,7 @@ export const defaultI18nResources: Resource = {
             // RePrompts
             MULTI_VALUE_LIST_CONTROL_DEFAULT_REPROMPT_VALUE_ADD: 'OK, added {{value}}.',
             MULTI_VALUE_LIST_CONTROL_DEFAULT_REPROMPT_VALUE_REMOVE: 'OK, removed {{value}}.',
-            MULTI_VALUE_LIST_CONTROL_DEFAULT_REPROMPT_VALUE_CLEARED: 'OK, removed {{value}} from the list.',
+            MULTI_VALUE_LIST_CONTROL_DEFAULT_REPROMPT_VALUE_CLEARED: 'OK, cleared {{value}} from the list.',
             MULTI_VALUE_LIST_CONTROL_DEFAULT_REPROMPT_ACTION_SUGGEST:
```

Comment:
As part of this, please update the names of the constants on :173-:176 to match the style of others.

Reply:
Fixed on latest commit :)